### DISCUSSION TITLE
Background in Covers cards

### DISCUSSION THREAD

**Discussion by MyWebmasterD** [Clooos/Bubble-Card#960] [Feature requests]
Hello,
I always thinked it was a good a idea to add a color background in the cover cards, to visualize the % of the open state, like a button slider

**Comment by Clooos** [comment]
Hi! I'm not planning to add this feature for now, but you can create a slider button with 3 sub-buttons, then you can change their tap actions to cover.open_cover, cover.stop_cover and cover.close_cover. This way it will be exactly how you want it 🙂

**Reply by MyWebmasterD** [reply]
I tried but the slider was interfering with the buttons...can I disable the touch slider?

**Reply by Clooos** [reply]
I don't have my computer with me right now but it should be possible to deactivate it with a custom style.

You can also take a look at this:
https://github.com/Clooos/Bubble-Card/discussions/928

**Reply by MyWebmasterD** [reply]
thank you, will try!
